tests/baselines/reference/importHelpersInTsx.js

```diff
@@ -0,0 +1,26 @@
+//// [tests/cases/compiler/importHelpersInTsx.tsx] ////
+
+//// [external.tsx]
+declare var React: any;
+declare var o: any;
+export const x = <span {...o} />
+
+//// [internal.tsx]
+declare var React: any;
+declare var o: any;
+const x = <span {...o} />
+
+//// [external.js]
+"use strict";
+var tslib_1 = require("tslib");
+exports.x = React.createElement("span", tslib_1.__assign({}, o));
+//// [internal.js]
+var __assign = (this && this.__assign) || Object.assign || function(t) {

```

Comment:
External modules can import "tslib". Scripts have no mechanism to ensure that "tslib" is loaded. If you want to completely suppress emit helpers, in this case you need to use both `--importHelpers` and `--noEmitHelpers`.